content/pl/port-binding.md

```diff
@@ -0,0 +1,14 @@
+## VII. Przydzielanie portów
+### Eksportuj usługi przez przydzielenie portów
+
+Aplikacje internetowe czasem uruchamiane są wewnętrz webserwera.  Aplikacje napisane w PHP np. działają jako moduł wewnątrz [Apache HTTPD](http://httpd.apache.org/), natomiast aplikacje w Javie mogą być uruchomione wewnątrz [Tomcata](http://tomcat.apache.org/).
+
+**Aplikacja dwunastu aspektów jest zupełnie niezależna** i może działać bez wstrzykiwania jej jako zależności w środowisku webserwera. Aplikacja internetowa **eksportuje HTTP jako usługę przez przydzielenie portu** i nasłuchuje zapytań przychodzących do niego zapytań.
```

Comment:
1. `wstrzykiwania jej jako zależności w środowisku webserwera` to totalnie nie brzmi po polsku.
2. `eksportuje` - `udostępnia` albo coś innego, byle konsekwentnie z tytułem 
3. `i nasłuchuje **zapytań** przychodzących do niego **zapytań**.`